addons/l10n_es_pos/static/tests/tours/spanish_pos_tour.js

```diff
@@ -57,14 +57,16 @@ registry.category("web_tour.tours").add("spanish_pos_tour", {
 });
 
 registry.category("web_tour.tours").add("l10n_es_pos_settle_account_due", {
-    steps: () =>
-        [
+    steps: () => {
+        const todayYear = luxon.DateTime.local().toFormat("yyyy");
```

Comment:
Why do we need to use a library to get the current year?
```suggestion
        const todayYear = (new Date()).getFullYear();
```